mm2src/coins/rpc_command/account_balance.rs

```diff
@@ -0,0 +1,111 @@
+use crate::coin_balance::HDAddressBalance;
+use crate::hd_wallet::HDWalletCoinOps;
+use crate::rpc_command::hd_account_balance_rpc_error::HDAccountBalanceRpcError;
+use crate::{lp_coinfind_or_err, CoinBalance, CoinWithDerivationMethod, MmCoinEnum};
+use async_trait::async_trait;
+use common::mm_ctx::MmArc;
+use common::mm_error::prelude::*;
+use common::PagingOptionsEnum;
+use crypto::{Bip44Chain, RpcDerivationPath};
+use std::fmt;
+
+#[derive(Deserialize)]
+pub struct HDAccountBalanceRequest {
+    coin: String,
+    #[serde(flatten)]
+    params: AccountBalanceParams,
+}
+
+#[derive(Deserialize)]
+pub struct AccountBalanceParams {
+    pub account_index: u32,
+    pub chain: Bip44Chain,
```

Comment:
In UTXO, `chain` means either `External` or `Internal` address.
* Receiving = 0 - Addresses that we will give out to people for receiving payments.
* Change = 1 - Addresses we use for sending change back to ourselves when we make transactions.
It can't be preconfigured for coins.

Reply:
Oh, I see, thanks for explanation :)